### PR TITLE
Update angular.md

### PR DESCRIPTION
**Pull request by yashints**
Added a note to mention the prefetching would not work in dev mode

### CODE REVIEW

File: content/angular.md [hunk 1]
```diff
@@ -297,6 +297,8 @@ When the user navigates from "Home" to "Example," the browser provides the "Exam
 
 Same happens when the user goes from "About" to "Media" since Guess.js prefetches the "Media" bundles when the user initially visits "About."
 
+**Note**: The prefecthing would not work in `dev` mode. You should build the application in production mode to see the prefetching working.
```
Comment (mgechev) [inline content/angular.md]: Would you add the command for this:

```
<div style="background-color: #e7f4ff; border-radius: 5px; padding: 20px; margin-bottom: 20px;>
<strong>Note:</strong> The prefecthing would not work in dev mode. You should build the application in production mode to see the prefetching working. To try it out run: <code>npm run build</code>
</div>
```

Comment (mgechev) [reply content/angular.md]: The snippet above also suggests an outline.

Comment (yashints) [reply content/angular.md]: Sure, BTW npm run build won't build for production, would it without --prod flag?

Comment (yashints) [reply content/angular.md]: Done, I was going to add something for windows as well, but thought let's see how I go about it tonight 🙃

**Review by mgechev — APPROVED** [review]

**Pull request merged**